lib/rubocop/cop/performance/sample.rb

```diff
@@ -89,7 +89,6 @@ def method_arg
             arg.source if arg
           end
 
-          # FIXME: use Range#size once Ruby 1.9 support is dropped
```

Comment:
I'm not sure about this. It essentially makes us `eval` the source, which feels a bit iffy. We could potentially use a capture and funcall (?), but the entire body of the cop has been wrapped in an inner class (and while I appreciate the intentions, I'm pretty sure it's the wrong abstraction 😅), which means we can't use the node pattern macros.

Furthermore, this should probably go in a `RangeNode` mixin or similar.